src/framework/glslang_path.ts

```diff
@@ -0,0 +1,12 @@
+import { assert } from './util/index.js';
+
+let glslangPath: string | undefined;
+
+export function getGlslangPath(): string | undefined {
+  return glslangPath;
+}
+
+export function setGlslangPath(path: string): void {
+  assert(path.startsWith('/'), 'glslang path must be absolute');
```

Comment:
Will this work on Windows?

Reply:
Yes, on web it's an absolute http request path, and on node it's never needed because glslang is in a known location.